### DISCUSSION TITLE
Open for new Maintainers?

### DISCUSSION THREAD

**Discussion by henrysachs** [supabase/supabase#16019] [Questions]
Hi there,

I just wanted to use the Helm Chart for Supabase and encountered more than one issue with it. It seems like there aren't that frequent contributions (which is fine because its community driven!). So I wanted to ask if its able to get me to the maintainers list so i can add some contributions or what would be the process? :)

**Comment by aantti** [comment]
Not sure if you are still interested in this topic, but if you are, I'd be happy to discuss.

**Comment by aantti** [comment]
Maintenance for Helm chart restarted in Dec 2025 - with @luizfelmach  as the key maintainer and contributor.
